include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 11
+#define SOF_ABI_MINOR 12
```

Comment:
there are multiple patches that update the ABI_MINOR, it'd be good to have consistency on what ABI_MINOR 12 is. @lgirdwood it seems your classifier is not being used?

Reply:
> there are multiple patches that update the ABI_MINOR, it'd be good to have consistency on what ABI_MINOR 12 is. @lgirdwood it seems your classifier is not being used?

@plbossart I thought only @lgirdwood adds stuff to the clasifier. If that is free for all we could put this change there and move on :)

Reply:
@groncarolo @plbossart OK, will skip updating the ABI with this patch and will piggy on ABI_MINOR 12 when it will be updated. I have updated the classifier table.

@groncarolo please update this patch by removing the ABI change.